examples/chip-tool/commands/clusters/OnOff/On.h

```diff
@@ -19,8 +19,6 @@
 #ifndef __CHIPTOOL_ON_H__
 #define __CHIPTOOL_ON_H__
 
-#include "common/ModelCommand.h"
-
 class On : public ModelCommand
```

Comment:
why was the include removed? I think we need all headers to include what they need an ModelCommand seems to be defined somewhere else.

Reply:
I finally moved all the files under a specific cluster inside Commands.h. I just dig a big cluster in #2792 and that was way too painful to add a file per command. So the include is only in ClusterName/Commands.h